test/unit-tests/GAC.t.sol

```diff
@@ -0,0 +1,222 @@
+        gac.setMultiMessageReceiver(DST_CHAIN_ID, address(42));
+    }
+
+    /// @dev cannot set multi message receiver to zero address
+    function test_set_multi_message_receiver_zero_address() public {
+        vm.startPrank(owner);
+
+        vm.expectRevert(Error.ZERO_ADDRESS_INPUT.selector);
+        gac.setMultiMessageReceiver(DST_CHAIN_ID, address(0));
+    }
+
+    /// @dev cannot set multi message receiver on zero chain ID
+    function test_set_multi_message_receiver_zero_chain_id() public {
+        vm.startPrank(owner);
+
+        vm.expectRevert(Error.ZERO_CHAIN_ID.selector);
+        gac.setMultiMessageReceiver(0, address(42));
+    }
+
+    /// @dev sets global message delivery gas limit
+    function test_set_global_msg_delivery_gas_limit() public {
+        vm.startPrank(owner);
+
+        uint256 oldLimit = gac.getGlobalMsgDeliveryGasLimit();
+        vm.expectEmit(true, true, true, true, address(gac));
+        emit DstGasLimitUpdated(oldLimit, 420000);
+
+        gac.setGlobalMsgDeliveryGasLimit(420000);
+
+        assertEq(gac.getGlobalMsgDeliveryGasLimit(), 420000);
+    }
+
+    /// @dev only owner can set global message delivery gas limit
+    function test_set_global_msg_delivery_gas_limit_only_owner() public {
+        vm.startPrank(caller);
+
+        vm.expectRevert("Ownable: caller is not the owner");
+        gac.setGlobalMsgDeliveryGasLimit(420000);
+    }
+
+    /// @dev sets message expiry time
+    function test_set_msg_expiry_time() public {
+        vm.startPrank(owner);
+
+        gac.setMsgExpiryTime(42000);
+
+        assertEq(gac.getMsgExpiryTime(), 42000);
+    }
+
+    /// @dev only owner can set message expiry time
+    function test_set_msg_expiry_time_only_owner() public {
+        vm.startPrank(caller);
+
+        vm.expectRevert("Ownable: caller is not the owner");
+        gac.setMsgExpiryTime(42000);
+    }
+
+    /// @dev cannot set message expiry time to zero
+    function test_set_msg_expiry_time_zero() public {
+        vm.startPrank(owner);
+
+        vm.expectRevert(Error.ZERO_EXPIRATION_TIME.selector);
+        gac.setMsgExpiryTime(0);
+    }
+
+    /// @dev sets refund address
+    function test_set_refund_address() public {
+        vm.startPrank(owner);
+
+        gac.setRefundAddress(address(42));
+
+        assertEq(gac.getRefundAddress(), address(42));
+    }
+
+    /// @dev only owner can set refund address
+    function test_set_refund_address_only_owner() public {
+        vm.startPrank(caller);
+
+        vm.expectRevert("Ownable: caller is not the owner");
+        gac.setRefundAddress(address(42));
+    }
+
+    /// @dev cannot set refund address to zero
+    function test_set_refund_address_zero() public {
+        vm.startPrank(owner);
+
+        vm.expectRevert(Error.ZERO_ADDRESS_INPUT.selector);
+        gac.setRefundAddress(address(0));
+    }
+
+    /// @dev only owner is privileged caller
+    function test_is_privileged_caller() public {
```

Comment:
Given `isPrivilegedCaller()` just checks if the provided address is the owner and we refer to the corresponding getter `getGlobalOwner()`, it would be less confusing to call this function `isGlobalOwner()` or `isOwner()` instead

Reply:
Done.